node/coinstacks/gnosis/indexer/config.json

```diff
@@ -0,0 +1,21 @@
+{
+  "fiat_rates": "coingecko",
+  "fiat_rates_params": "{\"url\": \"https://api.coingecko.com/api/v3\", \"coin\": \"gnosis\",\"platformIdentifier\": \"gnosis\",\"platformVsCurrency\": \"gno\",\"periodSeconds\": 900}",
+  "fiat_rates_vs_currencies": "AED,ARS,AUD,BDT,BHD,BMD,BRL,CAD,CHF,CLP,CNY,CZK,DKK,EUR,GBP,HKD,HUF,IDR,ILS,INR,JPY,KRW,KWD,LKR,MMK,MXN,MYR,NGN,NOK,NZD,PHP,PKR,PLN,RUB,SAR,SEK,SGD,THB,TRY,TWD,UAH,USD,VEF,VND,ZAR,BTC,ETH",
+  "mempoolTxTimeoutHours": 24,
+  "queryBackendOnMempoolResync": false,
+  "coin_name": "Gnosis",
+  "coin_shortcut": "GNO",
+  "coin_label": "Gnosis",
+  "rpc_url": "ws://localhost:8545",
```

Comment:
i was also seeing some path routing in the node config `--help` with separate port have you confirmed `:8546` connects to websocket as expected?
```suggestion
  "rpc_url": "ws://localhost:8546",
```

Reply:
![Uploading Screenshot from 2023-05-09 14-19-34.png…]()

Reply:
![Screenshot from 2023-05-09 14-19-34](https://github.com/shapeshift/unchained/assets/5132069/a02c9915-bfc3-4f75-a684-cd71b7368019)